### PR TITLE
added docs from archive

### PR DESCRIPTION
**Pull request by maxant**
Hi, as requested, here is a PR for the docs. I've copied pretty much everything I could find from the archive and migrated it over to asciidoc.  You can generate the HTML out of it using `mvn generate-resouces`. Hope it meets your expectations!

Cheers,
Ant

### CODE REVIEW

File: btm-docs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <artifactId>btm-docs</artifactId>
+    <name>Bitronix Transaction Manager :: Docs</name>
+    <packaging>pom</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.asciidoctor</groupId>
+                <artifactId>asciidoctor-maven-plugin</artifactId>
+                <version>1.5.2.1</version>
```
Comment (lorban) [inline btm-docs/pom.xml]: This version should be specified in the parent pom's pluginManagement section.

File: btm-docs/pom.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                        <id>output-html</id> 
+                        <phase>generate-resources</phase> 
+                        <goals>
+                            <goal>process-asciidoc</goal> 
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <sourceDirectory>src/main/doc</sourceDirectory>
+                    <outputDirectory>target/docs</outputDirectory>
```
Comment (lorban) [inline btm-docs/pom.xml]: Both folders should be prefixed with ${basedir}/ for peace of mind.

**Comment by lorban** [comment]
Why did you name the files *.txt rather than *.adoc?

**Comment by lorban** [comment]
Don't you think it would be a good idea to add this btm-doc module to the parent pom's <modules> section?

File: btm-docs/pom.xml (hunk 1 above)
Comment (maxant) [inline btm-docs/pom.xml]: done

File: btm-docs/pom.xml (hunk 2 above)
Comment (maxant) [inline btm-docs/pom.xml]: done

**Comment by maxant** [comment]
I've addressed the issues. The parent POM already referenced the new module. Files have been renamed and moved to default folders.  Pushed and this pull request now reflects the new changes. Let me know if anything else needs doing.

**Pull request merged**